lms/templates/dashboard/_dashboard_credit_info.html

```diff
@@ -20,18 +20,21 @@
     % elif not credit_status["purchased"]:
         <p class="credit-eligibility-msg">
             % if credit_status["deadline"] < datetime.datetime.now(pytz.UTC) + datetime.timedelta(days=30):
-                ${_("The opportunity to purchase credit for this course expires on {deadline}. You've worked hard - don't miss out!").format(deadline=get_default_time_display(credit_status["deadline"]))}
+                ${_("Your eligibility to receive {provider_name} credit is expiring on {deadline}. Don't miss out!").format(
+                    provider_name=credit_status["provider_name"],
+                    deadline=get_default_time_display(credit_status["deadline"]),
+                )}
             % else:
-                ${_("Congratulations - you have met the requirements for credit in this course!")}
+                ${_("You are now eligible for {provider_name} Credit. Congratulations!").format(provider_name=credit_status["provider_name"])}
             % endif
         </p>
         <div class="purchase_credit">
             ## TODO: set the URL for the link to the provider selection page on the E-Commerce service
-            <a class="btn purchase-credit-btn" href="#" target="_blank" data-course-key="${credit_status["course_key"]}">${_("Purchase Course Credit")}</a>
+            <a class="btn credit-btn purchase-credit-btn" href="#" target="_blank" data-course-key="${credit_status["course_key"]}">${_("Get credit")}</a>
         </div>
     % elif credit_status["request_status"] in [None, "pending"]:
```

Comment:
@srpearce FYI we are going to need one more message for credit eligibility on the dashboard.  The case is that the user has paid for credit, but still needs to start the "identity verification process" on the credit provider's site.